rbkit-lib/ui/cpuview.cpp

```diff
@@ -0,0 +1,62 @@
+#include "cpuview.h"
+#include "ui_cpuview.h"
+#include <QDebug>
+#include "model/cpustorage.h"
+
+CpuView::CpuView(QWidget *parent) :
+    QWidget(parent),
+    ui(new Ui::CpuView)
+{
+    callGraphModel = new QStandardItemModel;
+    flatGraphModel = new QStandardItemModel;
+
+    ui->setupUi(this);
+
+    connect(this,
+            SIGNAL(fillCallGraph(QStandardItemModel*)),
+            RBKit::CpuStorage::getStorage().data(),
+            SLOT(fillCallGraphModel(QStandardItemModel*)));
+
+    connect(this,
+            SIGNAL(fillFlatProfile(QStandardItemModel*)),
+            RBKit::CpuStorage::getStorage().data(),
+            SLOT(fillFlatProfileModel(QStandardItemModel*)));
+
+    // generating both the models
+    emit fillCallGraph(callGraphModel);
```

Comment:
because when we click on the `stop cpu profiling` button
it emits a signal to create a new cpu view
so we need to get the model that time
because there is no other button/interaction mechanism
that the user should do for tree to fill up